save-common/src/commonMain/kotlin/org/cqfn/save/core/plugin/Plugin.kt

```diff
@@ -116,7 +116,7 @@ abstract class Plugin(
                     }
                 }
             }.toList()
-            val notFoundTests = testFiles.filter { it !in foundTests.map { foundTest -> foundTest.test.toString() } }
+            val notFoundTests = testFiles.filter { testFile -> testFile !in foundTests.map { foundTest -> foundTest.test.toString() } }
```

Comment:
maybe it will be good to have logic on number of maximum inline lambdas in diktat

Reply:
Yes, I agree. This logic could be added into `NewlinesRule` as a new inspection. Feel free to open an issue in diktat.